src/pages/components/tipSection/index.js

```diff
@@ -31,17 +60,17 @@ const TipSection = () => {
         </div>
 
         {/* Button */}
-        <div className='flex flex-1 flex-col justify-center'>
-          <div className='flex justify-center pb-[30px] sm:pb-[55px]'>
-            <button className='h-[38px] w-20 rounded-l-lg bg-navy-600 px-2 py-1 text-lg font-semibold text-white hover:underline dark:bg-orange-900'>
+        <div className='justify-top flex flex-1 flex-col '>
+          <div className='flex justify-center pb-[30px] pt-4 sm:pb-[35px]'>
+            <button className={`${isActive}`} onClick={() => setIsDemo(false)}>
               {t('tipsBtnOne')}
             </button>
-            <button className='h-[38px] w-20 rounded-r-lg bg-navy-100 px-2 py-1 text-lg font-semibold text-dark-ash-900 hover:underline dark:bg-orange-200'>
+            <button className={`${isNotActive}`} onClick={() => setIsDemo(true)}>
               {t('tipsBtnTwo')}
             </button>
           </div>
           <div className='flex flex-col items-center md:px-14'>
-            <Tips />
+            {isDemo === true ? <Demo /> : <Tips />}
```

Comment:
```suggestion
            {isDemo ? <Demo /> : <Tips />}
```

@CaneteJoash here as well